### PR TITLE
fix: rename as_bytes to as_byte in ReadableSize (#428)

### LINKED ISSUE #428: Bad name in ReadableSize
### Describe this problem

https://github.com/CeresDB/ceresdb/blob/659b55985a14f835cff5bbb2df4eea1b0a540366/common_util/src/config.rs#L149

In `ReadableSize`, there is a method called `as_bytes()`, which is very confusing, since the original meaning of `as_bytes` is to return `&[u8]`, not `u64`

### Steps to reproduce

No required.

### Expected behavior

_No response_

### Additional Information

I suggest we name it `as_byte` to follow other methods in ReadableSize

### PR DESCRIPTION
**Pull request by zouxiang1993**
## Which issue does this PR close?

Closes #428

### CODE REVIEW

**Comment by ShiKaiWi** [comment]
@zouxiang1993 Thanks for your contribution. It looks pretty and will be merged when the status check finishes.

File: server/src/server.rs [hunk 1]
```diff
@@ -355,26 +355,29 @@ impl<Q: QueryExecutor + 'static> Builder<Q> {
...
-            .cluster(self.cluster.clone())
-            .opened_wals(opened_wals)
-            .schema_config_provider(provider)
-            .forward_config(self.server_config.forward)
-            .timeout(self.server_config.timeout.map(|v| v.0))
-            .auto_create_table(self.server_config.auto_create_table)
-            .build()
-            .context(BuildGrpcService)?;
+        let rpc_services =
+            grpc::Builder::new()
```
Comment (chunshao90) [inline server/src/server.rs:359]: No need to change here?

Comment (zouxiang1993) [reply server/src/server.rs:359]: "cargo fmt" do it.

**Review by ShiKaiWi — APPROVED** [review]
LGTM

**Pull request merged**